### PR TITLE
7 merchant dashboard invoices sorted by least recent

### LINKED ISSUE #7: Merchant Dashboard Invoices sorted by least recent
Merchant Dashboard Invoices sorted by least recent

As a merchant
When I visit my merchant dashboard
In the section for "Items Ready to Ship",
Next to each Item name I see the date that the invoice was created
And I see the date formatted like "Monday, July 18, 2019"
And I see that the list is ordered from oldest to newest

### PR DESCRIPTION
**Pull request by rebeckahendricks**
*Reorganize ready_to_ship method by moving it to the Item class.
*Add AR order to order items ready to ship by their invoice created_at date.
****Issue in creating full tests for model and feature due to lack of diversity in test db****

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -72,6 +72,13 @@ GEM
     docile (1.4.0)
     erubi (1.10.0)
     execjs (2.8.1)
+    factory_bot (6.2.1)
```
Comment (thayes87) [inline Gemfile.lock:75]: Interested to see how this works. Did you use the Gist from our "What's a Dev?"

**Review by lcole37 — APPROVED** [review]

File: app/models/customer.rb [hunk 2]
```diff
@@ -10,7 +10,12 @@ class Customer < ApplicationRecord
 
 
   def self.top_five_customers_by_merchant
-    joins(:transactions).where('transactions.result = 0').group(:last_name, :first_name).order(count: :desc).limit(5).count
+    joins(:transactions)
```
Comment (thayes87) [inline app/models/customer.rb:13]: I like how you lined up the "." here to make the code more readable

File: Gemfile.lock (hunk 1 above)
Comment (Dominicod) [reply Gemfile.lock:75]: Interested in this as well

**Review by thayes87 — APPROVED** [review]
your getting good at Active Record whether you want to hear it or not

File: app/models/customer.rb (hunk 2 above)
Comment (Dominicod) [reply app/models/customer.rb:13]: Same! Was wondering if you are able to do that for the admin version as well?

**Review by Dominicod — APPROVED** [review]
Looks good!

**Pull request merged**